libraries/AP_Notify/RGBLed.h

```diff
@@ -79,32 +78,32 @@ class RGBLed: public NotifyDevice {
     ((S0) << (0*3) | (S1) << (1*3) | (S2) << (2*3) | (S3) << (3*3) | (S4) << (4*3) | (S5) << (5*3) | (S6) << (6*3) | (S7) << (7*3) | (S8) << (8*3) | (S9) << (9*3))
 
 #define DEFINE_COLOUR_SEQUENCE_SLOW(colour)                       \
-    DEFINE_COLOUR_SEQUENCE(colour,colour,colour,colour,colour,OFF,OFF,OFF,OFF,OFF)
+    DEFINE_COLOUR_SEQUENCE(colour,colour,colour,colour,colour,BLACK,BLACK,BLACK,BLACK,BLACK)
 #define DEFINE_COLOUR_SEQUENCE_FAILSAFE(colour) \
     DEFINE_COLOUR_SEQUENCE(YELLOW,YELLOW,YELLOW,YELLOW,YELLOW,colour,colour,colour,colour,colour)
 #define DEFINE_COLOUR_SEQUENCE_SOLID(colour) \
     DEFINE_COLOUR_SEQUENCE(colour,colour,colour,colour,colour,colour,colour,colour,colour,colour)
 #define DEFINE_COLOUR_SEQUENCE_ALTERNATE(colour1, colour2)                      \
     DEFINE_COLOUR_SEQUENCE(colour1,colour2,colour1,colour2,colour1,colour2,colour1,colour2,colour1,colour2)
 
-#define OFF    0
```

Comment:
I could've swarn these were undef'd after use.

Reply:
These really should probably just be an enum class here that the macro then uses and abuses rather then defines, but I wasn't gonna stick peter with that....